backend/store/migration/prod/LATEST.sql

```diff
@@ -709,7 +709,7 @@ CREATE TABLE instance_change_history (
     created_ts BIGINT NOT NULL DEFAULT extract(epoch from now()),
     updater_id INTEGER NOT NULL REFERENCES principal (id),
     updated_ts BIGINT NOT NULL DEFAULT extract(epoch from now()),
-    instance_id INTEGER NOT NULL REFERENCES instance (id),
+    instance_id INTEGER REFERENCES instance (id),
```

Comment:
Add a comment here otherwise it's weird that instance_change_history allows NULL instance_id

Reply:
Done.